lib/streamlit/runtime/fragment.py

```diff
@@ -354,14 +353,14 @@ def fragment(
     >>>     st.session_state.app_runs = 0
     >>>     st.session_state.fragment_runs = 0
     >>>
-    >>> @st.experimental_fragment
-    >>> def fragment():
+    >>> @st.fragment
+    >>> def my_fragment():
```

Comment:
heads up @sfc-gh-dmatthews -- I made some minor changes to the docstrings in this file. 

Most notably, I removed the `experimental_` prefix throughout and changed the name of this function from `fragment` to `my_fragment` since it felt a bit weird to have the function and decorator share the same name (even though it doesn't really matter since the decorator is namespaced).

Reply:
Noted. Thanks.